runtime/doc/various.txt

```diff
@@ -211,6 +211,11 @@ g8			Print the hex values of the bytes used in the
 :te[rminal][!]		Open an interactive shell in a |terminal-emulator|.
 			Use the shell specified in 'shell'.
 
+			The new terminal buffer takes over the current split
```

Comment:
Ok, I'll `s/split window/window`

Happy to remove the stuff about failing if the current buffer has changes. I thought it was worthwhile at least having a go at documenting the meaning of `[!]` in `:terminal!`, but there's a lot of complexity there and to document it thoroughly would mean adding lots and lots of words.